src/addin/addin-client.ts

```diff
@@ -406,6 +406,11 @@ export class AddinClient {
               this.args.callbacks.buttonClick();
             }
             break;
+          case 'update-context':
+            if (this.args.callbacks.updateContext) {
+              this.args.callbacks.updateContext();
```

Comment:
And then pass in `data.message` to updateContext. `this.args.callbacks.updateContext(data.message);`

Reply:
resolved